cfme/services/catalogs/__init__.py

```diff
@@ -9,6 +9,8 @@
 
 
 class ServicesCatalogView(BaseLoggedInPage):
+
+    @property
```

Comment:
**REQUIRED:** Check usage against https://github.com/ManageIQ/integration_tests/blob/master/cfme/services/catalogs/ansible_catalog_item.py#L155

Reply:
@psav fixed and checked, everything works fine.